dkms.in

```diff
@@ -833,6 +816,90 @@ run_build_script() {
     fi
 }
 
+# helper functions for getting cc/ld used by kernel
```

Comment:
Do you mind adding a bit more context here;
Create an array of candidates, first one having the complete version suffixed and strip numbers until we find a match.